apps/api-extractor/.vscode/launch.json

```diff
@@ -4,6 +4,19 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
Just moving this around for funzies?

Reply:
The profiles are easier to find when they appear at the top of the list.